### DISCUSSION TITLE
Ability to disable dot and bracket syntax in field names

### DISCUSSION THREAD

**Discussion by SethTompkins** [react-hook-form/react-hook-form#1798] [Ideas] [answered]
I have a few projects where the API data uses key-value pairs that have dots in the keys. It would be great if we could have an option to pass to `register` that would disabled the dot/bracket syntax inference on a per-field basis.

**Comment by bluebill1049** [comment, marked as answer]
that's not possible at this stage, we need the`.` and `[]` to identify object and array.

**Reply by mariorodriguespt** [reply]
We could use a prop to disable dot syntax, fairly annoying to rename fields twice to get it working right when you have dynamic field name with dots.

**Comment by bombillazo** [comment]
It would be ideal if the library allowed for custom delimiters to parse object/arrays from name path.
